File: Arrays Python/even_numbers.py
def even_numbers(numbers):
	
	even_count = 0
	for index in range(len(numbers)):
		if numbers[index] % 2 == 0:
			even_count+=1

	even_numbers = [0] * even_count
	index_counting = 0

	for count in range(len(numbers)):
		if numbers[count] % 2 == 0:
			even_numbers[index_counting] = numbers[count]
			index_counting+=1

	return even_numbers
even = [2, 7, 3, 8, 9, 6, 5, 4]
reult = even_numbers(even)
print(reult)

File: Arrays Java/EvenNumbers.java
import java.util.Arrays;
public class EvenNumbers{
	public static int[] checkEvenNumbers(int[] arrays){
	
		int evenCount = 0;
		for(int index = 0; index < arrays.length; index++){
			if(arrays[index] % 2 == 0){
				evenCount++;
			}
		}
		
		int[] evenNumbers = new int[evenCount];
		int count = 0;

		for(int index = 0; index < arrays.length; index++){
			if(arrays[index] % 2 == 0){
				evenNumbers[count] = arrays[index];
				count++;
			}
		}

		return evenNumbers;
	}
		

	public static void main(String[]args){
	
		int[] outPut = {4, 7, 8, 5, 6, 9, 4, 2, 1, 10, 3};
		System.out.println("The even numbers in the arrays are: " + Arrays.toString(EvenNumbers.checkEvenNumbers(outPut)));	
	}		
}